samples/tab-deeplink-all-features/nodejs/README.md

```diff
@@ -0,0 +1,119 @@
+---
+page_type: sample
+description: Microsoft Teams sample app for demonstrating deeplink all features using Tab
```

Comment:
Microsoft Teams sample app for demonstrating deep link features using tab

Reply:
fixed